README.md

```diff
@@ -32,6 +32,16 @@ further details.
 
 It's worth pointing out that packages such as [Images.jl](https://github.com/JuliaImages/Images.jl) load FileIO for you.
 
+Loading an image is then as simple as
+
+```julia
+img = load(filename[; view=false])
+```
+
+The optional `view` keyword argument is useful for reducing memory consumption
+when loading large files.
```

Comment:
yes, putting in on `ImageMagick.load` makes `?ImageMagick.load` work.  but given that one (or something) must do a `using ImageMagick` and that docstring is not displayed with `?load`, is it worth it?